recipes-kernel/linux/linux-nilrt_git.bb

```diff
@@ -8,5 +8,5 @@ require linux-nilrt.inc
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-SRCREV_xilinx-zynq = "4442d176ee60b472252436950058a85fce08a3a3"
-SRCREV_qemu-zynq7 = "4442d176ee60b472252436950058a85fce08a3a3"
+SRCREV_xilinx-zynq = "e167d2ccda8e812cd7aa99fefc59269c854952ce"
```

Comment:
Can you also update these hashes in `linux-nilrt-debug_git.bb`? And squash the commits?